anyway/widgets/road_segment_widgets/street_view_widget.py

```diff
@@ -16,10 +16,13 @@ def __init__(self, request_params: RequestParams):
 
     def generate_items(self) -> None:
         self.items = {
-            "longitude": self.request_params.gps["lon"],
-            "latitude": self.request_params.gps["lat"],
+            "longitude": self.request_params.gps.get("lon"),
+            "latitude": self.request_params.gps.get("lat"),
         }
 
+    def is_included(self):
+        return self.request_params.gps is not None
+
```

Comment:
Maybe it is safer to do
`return self.request_params.gps and self.request_params.gps.get("lon") and self.request_params.gps.get`

Reply:
thanks! fixed